### ISSUE TITLE
Link to the unlisted review page is missing for an unlisted add-on block

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-server#12797]
Follow up for #12452 

### Describe the problem and steps to reproduce it:
1. Log in to AMO with an admin account
2. Open the Admin tools and go to the Blocklist module
3. Add a new unlisted add-on to the block list - see [this](https://addons-internal.dev.mozaws.net/en-US/admin/models/blocklist/block/9/change/)
4. Once you land on the change block view, look at the field beneath 'Users' 

### What happened?
There is no link pointing to the unlisted version review page

### What did you expect to happen?
A link to the unlisted version review page should be present 

### Anything else we should know?
- listed add-on blocks do include a link to the review page

![image](https://user-images.githubusercontent.com/31961530/68187696-fb2e8380-ffaf-11e9-9176-a75eb60e3759.png)

**Comment by eviljeff** [comment]
btw, the `-` is what django admin renders when it gets an invalid value - normally when there's an exception thrown somewhere along the way.

**Comment by AlexandraMoga** [comment]
Verified fixed on -dev:

- pure unlisted:
![image](https://user-images.githubusercontent.com/31961530/68468339-7d24e380-0220-11ea-8c0a-9b50a469d7b7.png)


- mixed versions:
![image](https://user-images.githubusercontent.com/31961530/68468345-80b86a80-0220-11ea-90a8-a79a46bf9801.png)

Ideally, the "Review unlisted link" text should be present only if the add-on has actual unlisted versions, but this is a lower priority issue.

**Comment by eviljeff** [comment]
> Ideally, the "Review unlisted link" text should be present only if the add-on has actual unlisted versions, but this is a lower priority issue.

The link shows up for all addons even if they don't have unlisted versions?

**Comment by AlexandraMoga** [comment]
> The link shows up for all addons even if they don't have unlisted versions?

@eviljeff not the actual link, just the "Review unlisted link" text:

![image](https://user-images.githubusercontent.com/31961530/68471163-46ea6280-0226-11ea-8921-a8b6421fa7ae.png)
